src/main/java/uk/gov/companieshouse/insolvency/data/api/InsolvencyApiService.java

```diff
@@ -80,6 +87,15 @@ private ChangedResource mapChangedResource(String contextId,
         changedResource.setResourceKind("company-insolvency");
         changedResource.setContextId(contextId);
 
+        if (isDelete) {
+            try {
+                changedResource.setDeletedData(objectMapper.writeValueAsString(insolvencyDocument));
```

Comment:
I dont think we should serialise and send whole InsolvencyDocument in "deletedData", only CompanyInsolvency (or "data") within it

Reply:
i.e. the json data we include must match response coming back from existing /company/{company_number}/insolvency GET endpoint

Reply:
The insolvencyDocument is passed from the delete method where it first retrieves from db before delete operation.

Reply:
Correct. This should contain the response from the API of the data which has just been deleted.